specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/ResourceProvider.json

```diff
@@ -687,6 +687,10 @@
         "errorMessage": {
           "description": "Error message if unable to get token",
           "type": "string"
+        },
+        "systemData": {
```

Comment:
Sorry I should have clarified. Can you please add systemData to both "Resource" and "ProxyOnlyResource" definitions? That way all tracked & proxy resources will have it.

Reply:
Updated